device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-V256/pg_profile_lookup.ini

```diff
@@ -0,0 +1 @@
+../ACS-SN5600/pg_profile_lookup.ini
```

Comment:
We should not use the default `pg_profile_lookup.ini`.
The default one was calculated with `small packet percentage = 100%`. For MSFT, it should be 50%.
The buffer pool sizes need to be adjusted according to the new headroom size.